test/test_with_xpress.jl

```diff
@@ -439,6 +439,37 @@ end
     @test round(sum(r["ElectricStorage"]["soc_series_fraction"]), digits=2) / 8760 >= 0.7199
 end
 
+@testset "Solar and ElectricStorage w/ ElectricStorage cost constants" begin
```

Comment:
@toddleif I think you could remove all of these changes to `test_with_xpress.jl`. We don't have to update this file any more

Reply:
Done